spec/lib/jobs/upsert-node-job-spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+var uuid = require('node-uuid');
+
+describe('upsert-node-job', function(){
+    var job,
+    NodeUpsertor, 
+    taskId = uuid.v4(),
+    context = {},
+    options = {
+        target: "some:test:node",
+        data: { 
+            key1:{
+                ip: "123.456.789.012",
+                port: "12345",
+                key2: {
+                    key3: "value"
+                }
+            }
+        } 
+    },
+    sandbox = sinon.sandbox.create(),
```

Comment:
ohh, got it. I made a mistake, you are right.
I'd prefer add each variable declarative statement in a line if the variable is assigned some complex initial value, this looks more clear comparing with distinguish `,` and `;` . This may be my own preference, so not a must change for your pull request.

```javascript
var job, nodeUpsertor, foo=1, bar='abc';
var taskId = uuid.v4();
var options = {
     ...
};
var sandbox = sinon.sandbox.create();
var waterline = {};
```